bin/git-ignore-io

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+
+gitignore_io_url="https://www.gitignore.io/api/"
+
+default_path="$HOME/.gi_list"
+gi_list=`cat $default_path | tr "," "\n\n"`
```

Comment:
Oh, I was trying to use sed and forgot to change it back. I'll update latter. Thanks~